--- Makefile.extmod.orig	2023-04-19 10:04:57.088416000 +0300
+++ Makefile.extmod	2023-04-22 11:48:18.731220000 +0300
@@ -93,8 +93,7 @@
 HTTP_AUTH_KRB5_VARS=		DSO_EXTMODS+=auth_krb5
 HTTP_AUTH_KRB5_EXTRA_PATCHES=	${PATCHDIR}/extra-patch-spnego-http-auth-nginx-module-config
 
-HTTP_AUTH_LDAP_GH_TUPLE=	kvspb:nginx-auth-ldap:241200e:http_auth_ldap
-HTTP_AUTH_LDAP_EXTRA_PATCHES=	${PATCHDIR}/extra-patch-ngx_http_auth_ldap_module.c
+HTTP_AUTH_LDAP_GH_TUPLE=	freultwah:nginx-auth-ldap:9de312b:http_auth_ldap
 HTTP_AUTH_LDAP_VARS=		DSO_EXTMODS+=http_auth_ldap
 HTTP_AUTH_LDAP_USES=		ldap
 
--- distinfo.orig	2025-07-12 03:48:01.995564000 +0000
+++ distinfo	2025-07-13 09:06:26.714932000 +0000
@@ -43,8 +43,8 @@
 SIZE (TeslaGov-ngx-http-auth-jwt-module-2.3.1_GH0.tar.gz) = 32459
 SHA256 (stnoonan-spnego-http-auth-nginx-module-v1.1.3_GH0.tar.gz) = 0e1e54f8ab7d76f922881f13d870edf8b820ef645b19013774c825110d011640
 SIZE (stnoonan-spnego-http-auth-nginx-module-v1.1.3_GH0.tar.gz) = 32975
-SHA256 (kvspb-nginx-auth-ldap-241200e_GH0.tar.gz) = d3aa3a5ead311b34059d6ab1b6d2cf3ac8269c47a87b768523c07381229db470
-SIZE (kvspb-nginx-auth-ldap-241200e_GH0.tar.gz) = 18541
+SHA256 (freultwah-nginx-auth-ldap-9de312b_GH0.tar.gz) = 3c53bacbb3fb4f7f59cde514738dcbe890738268e20a91bd822211d812f8a28f
+SIZE (freultwah-nginx-auth-ldap-9de312b_GH0.tar.gz) = 18935
 SHA256 (sto-ngx_http_auth_pam_module-v1.5.5_GH0.tar.gz) = 98a71617d9119ae784993e3789ce8766fdf2ff2479691f3dc6cf8d8763f8d364
 SIZE (sto-ngx_http_auth_pam_module-v1.5.5_GH0.tar.gz) = 7233
 SHA256 (arut-nginx-dav-ext-module-v3.0.0_GH0.tar.gz) = d2499d94d82d4e4eac8425d799e52883131ae86a956524040ff2fd230ef9f859
