diff --git a/www/nginx-devel/Makefile b/www/nginx-devel/Makefile
index 1100f6e..5622765 100644
--- a/www/nginx-devel/Makefile
+++ b/www/nginx-devel/Makefile
@@ -201,6 +201,8 @@ STREAM_SSL_PREREAD_CONFIGURE_ON=--with-stream_ssl_preread_module
 
 .include <bsd.port.pre.mk>
 
+MONGOC_VERSION!=	${MAKE} -C ${PORTSDIR}/devel/mongo-c-driver -V DISTVERSION
+
 .if ${PORT_OPTIONS:MDSO}
 _addbasemod=	=dynamic
 _addextmod=	add-dynamic-module
diff --git a/www/nginx-devel/Makefile.extmod b/www/nginx-devel/Makefile.extmod
index 43dc9fd..80e2fe9 100644
--- a/www/nginx-devel/Makefile.extmod
+++ b/www/nginx-devel/Makefile.extmod
@@ -65,7 +65,6 @@ GRIDFS_GH_TUPLE=	nieoding:nginx-gridfs:059bdc3:gridfs
 GRIDFS_LIB_DEPENDS=	libbson2.so:devel/libbson \
 			libmongoc2.so:devel/mongo-c-driver
 GRIDFS_VARS=		DSO_EXTMODS+=gridfs GRIDFS_SUBDIR=/nginx-gridfs
-MONGOC_VERSION!=	cd ${.CURDIR}/../../devel/mongo-c-driver; make -V DISTVERSION
 GRIDFS_EXTRA_PATCHES=	${PATCHDIR}/extra-patch-ngnx-gridfs_ngx_http_gridfs_module.c \
 			${PATCHDIR}/extra-patch-nginx-gridfs_config
 
diff --git a/www/nginx/Makefile b/www/nginx/Makefile
index 23911b3..319ae76 100644
--- a/www/nginx/Makefile
+++ b/www/nginx/Makefile
@@ -195,6 +195,8 @@ STREAM_SSL_PREREAD_CONFIGURE_ON=--with-stream_ssl_preread_module
 
 .include <bsd.port.pre.mk>
 
+MONGOC_VERSION!=	${MAKE} -C ${PORTSDIR}/devel/mongo-c-driver -V DISTVERSION
+
 .if ${PORT_OPTIONS:MDSO}
 _addbasemod=	=dynamic
 _addextmod=	add-dynamic-module
diff --git a/www/nginx/Makefile.extmod b/www/nginx/Makefile.extmod
index 388ecda..51ec11a 100644
--- a/www/nginx/Makefile.extmod
+++ b/www/nginx/Makefile.extmod
@@ -65,7 +65,6 @@ GRIDFS_GH_TUPLE=	nieoding:nginx-gridfs:059bdc3:gridfs
 GRIDFS_LIB_DEPENDS=	libbson2.so:devel/libbson \
 			libmongoc2.so:devel/mongo-c-driver
 GRIDFS_VARS=		DSO_EXTMODS+=gridfs GRIDFS_SUBDIR=/nginx-gridfs
-MONGOC_VERSION!=	cd ${.CURDIR}/../../devel/mongo-c-driver; make -V DISTVERSION
 GRIDFS_EXTRA_PATCHES=	${PATCHDIR}/extra-patch-ngnx-gridfs_ngx_http_gridfs_module.c \
 			${PATCHDIR}/extra-patch-nginx-gridfs_config
 
